packages/xchain-client/src/BaseXChainClient.ts

```diff
@@ -1,6 +1,7 @@
 import { validatePhrase } from '@xchainjs/xchain-crypto'
 import { Asset, Chain } from '@xchainjs/xchain-util'
 import axios from 'axios'
+import { StdTx } from 'cosmos-client/x/auth'
```

Comment:
Don't import types from other client packages to avoid circular imports. If you need to do something like this, re-think about your structure. `xchain-client` is the top-most package in the hierarchy, which will be used (imported) by other packages. But not the the other way round.